### PR TITLE
Updated mec172xevb_assy6906 documentation

### PR DESCRIPTION
**Pull request by wu-frances**

### CODE REVIEW

**Review by albertofloyd — DISMISSED** [review]

**Review by tejlmand — DISMISSED** [review]
thanks for this contribution.

Some small comments regarding the build code.

**Comment by MaureenHelm** [comment]
Thank you for your contribution!

This PR does a lot more than updating board documentation. Please edit the PR title and description to reflect that you're adding a new board. Please also review the failed compliance check details - there are many checkpatch errors that need to be addressed, and the commit messages need to have a body.

File: boards/arm/mec172xmodular_assy6930/mec172xmodular_assy6930.dts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+};
+
+&cpu0 {
+	status = "okay";
+};
+
+/* Initialize ECIA. Does not initialize child devices */
+&ecia {
+	status = "okay";
+};
```
Comment (albertofloyd) [inline boards/arm/mec172xmodular_assy6930/mec172xmodular_assy6930.dts:55]: @wu-frances this is missing several eSPI related entries that should be supported  so modular card can be used with host. i.e. girq24, espi0, kbc0, acpi_ec0

**Comment by albertofloyd** [comment]
@wu-frances looks good, I assume you will be raising separate PR for modular card.
Do squash the commits.

File: boards/arm/mec172xmodular_assy6930/mec172xmodular_assy6930.dts (hunk 1 above)
Comment (albertofloyd) [reply boards/arm/mec172xmodular_assy6930/mec172xmodular_assy6930.dts]: PR scope changed, comment not applicable

**Comment by wu-frances** [comment]
Documentation for mec172xevb_assy6906 has been updated per comment.  The scope of the PR has been reduced to include just the MEC172x EVB documentation.  mec172xmodular_assy6930 board files and documentation will be submitted in a separate PR.

**Review by enjiamai — DISMISSED** [review]
For mec172xevb_assy6906 setup, it has verified works.

**Review by albertofloyd — DISMISSED** [review]

**Comment by tejlmand** [comment]
@wu-frances the commit message says:
> Original PR included mec172xmodular_assy6930 board files update.
Those files have been removed and will be included in separate
pull request.

This information is of no value once the PR is merged to main, so please remove those lines.

Extra info regarding commit messages can be seen here: https://docs.zephyrproject.org/latest/contribute/index.html#commit-message-body

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Please reduce the size of the images, some are over 1 MB

**Review by MaureenHelm — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]
lgtm - re-approved

**Pull request merged**